go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/NearNodeFlash/lustre-fs-operator
 
 go 1.19
 
+replace github.com/DataWorkflowServices/dws => ../dws
+
```

Comment:
You mentioned this, but just a reminder.

Reply:
Done